src/logic/Manager/Manager.ts

```diff
@@ -0,0 +1,177 @@
+import GameMap from 'logic/GameMap/GameMap';
+import { Messanger } from 'logic/Messanger/Messanger';
+import { Dice } from 'logic/Dice/Dice';
+import { PlayerObject } from 'logic/PlayerObject/PlayerObject';
+import { Direction } from 'logic/PlayerObject/types';
+import { Cell } from 'logic/Cell/Cell';
+import { IPlayer } from 'logic/IPlayer/IPlayer';
+
+export default class Manager {
+  private diceSize = 125;
+
+  private activePlayer: PlayerObject;
+
+  private cells: Cell[];
+
+  private playerObjects: PlayerObject[];
+
+  constructor(
+    private ctx: CanvasRenderingContext2D,
+    players: IPlayer[],
+    private map: GameMap,
+    private messanger: Messanger,
+    private onEndGame: () => void,
```

Comment:
🧐
Хорошо бы не мешать приватные и публичные методы класса
```suggestion
    private ctx: CanvasRenderingContext2D,
    private map: GameMap,
    private messanger: Messanger,
    private onEndGame: () => void,
    players: IPlayer[],
```